docs/.vuepress/theme/styles/config.styl

```diff
@@ -114,3 +118,11 @@ $contentWidthXL = 1440px;
 // code
 $lineNumbersWrapperWidth = 3.5rem
 $codeLang = js ts html md vue css sass scss less stylus go java c sh yaml py
+
+.reset-type
```

Comment:
This config.styl is really just for variables - I'd say put this class in theme.styl if you need a global class. It looks like this class is only used in SearchBox.vue though, why not just place it there?

Reply:
Removed it completely :)